src/helpers/strings.ts

```diff
@@ -48,5 +48,12 @@ export class Strings {
     static StatusCode401: string = "Unauthorized. Check your authentication credentials and try again.";
     static StatusCodeOffline: string = "It appears Visual Studio Code is offline.  Please connect and try again.";
     static ProxyUnreachable: string = "It appears the configured proxy is not reachable.  Please check your connection and try again.";
+
+    // TFVC messages/errors
+    static ChooseItemQuickPickPlaceHolder: string = "Choose a file to open the file.";
```

Comment:
"Choose a file to open..."... "it"?  "file to open the file" doesn't ready very well (to me).